pyiron_atomistics/dft/bader.py

```diff
@@ -26,22 +29,23 @@ class Bader:
     .. _Bader code: http://theory.cm.utexas.edu/henkelman/code/bader
     """
 
-    def __init__(self, job):
+    def __init__(self, structure, working_directory):
```

Comment:
if this now accepts a structure, working directory, instead of job, it will break previous jobs - I don't think anyone is using this code, so just a passing comment here on this change.